lib/util/git/modified-cache.ts

```diff
@@ -9,11 +8,15 @@ export function getCachedModifiedResult(
   const { branches } = getCache();
   const branch = branches?.find((branch) => branch.branchName === branchName);
 
-  if (branch?.sha !== branchSha) {
-    return null;
+  if (
+    branch?.sha &&
+    branch.sha === branchSha &&
```

Comment:
```suggestion
    branch?.sha === branchSha &&
```

isn't this the same? so if `sha` is missing, `isModified` is missing too? 🤔